profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Bernd Waibel <waebbl@gmail.com> (13 May 2019)
+# needs build and runtime testing
+>=media-gfx/alembic-1.7.9 arnold maya
```

Comment:
no need to constrain it to this min version. and no versions are currently stable anyway.